app/assets/javascripts/backbone/templates/timeline/item.jst.ejs

```diff
@@ -1,10 +1,12 @@
 <section class="<%= item.highlight ? 'highlight' : '' %>">
+  <% back_uri = encodeURIComponent(window.location.href); %>
   <div class="sectionRight">
-    <img onerror="image_loading_error(this);" src="<%= item.thumbnail %>" data-default-src="<%= window.default_img_path %>">
+    <a href="/item/<%= item.id %>?back_uri=<%= back_uri %>">
+      <img onerror="image_loading_error(this);" src="<%= item.thumbnail %>" data-default-src="<%= window.default_img_path %>"/>
+    </a>
   </div>
   <div class="sectionLeft">
     <h6><%= item.type %></h6>
-    <% back_uri = encodeURIComponent(window.location.href); %>
```

Comment:
I moved it b/c we are referencing the variable `back_uri` now in line 4 as well as line 10.